server/repository/odpDraft/odpDraftRepository.ts

```diff
@@ -1,14 +1,12 @@
 import { OdpClassRepository } from '@server/repository'
-import { insertAudit } from '../audit/auditRepository'
 
 export const getDraftId = async (client: any, odpId: any) => {
```

Comment:
this operates on ODP table. we might not need OdpDraftRepository.
A repository is a one to one correspondence with a db table